courses/free-courses-en.md

```diff
@@ -1234,6 +1236,7 @@
 * [Mathematics for Machine Learning - Linear Algebra](https://www.youtube.com/playlist?list=PLiiljHvN6z1_o1ztXTKWPrShrMrBLo5P3) - Imperial College London, Dr David Dye, Dr Sam Cooper
 * [Mathematics for Machine Learning - Multivariate Calclus](https://www.youtube.com/playlist?list=PLiiljHvN6z193BBzS0Ln8NnqQmzimTW23) - Imperial College London, Dr David Dye, Dr Sam Cooper
 * [Pattern Recognition and Machine Learning](https://www.microsoft.com/en-us/research/people/cmbishop/#!prml-book)
+* [Python Machine Learning & AI Mega Course ](https://www.youtube.com/playlist?list=PLBfoYdk-WAly5ib6M8ifOnDGl33bSvF_n) - Tech With Tim (Tim Ruscica)
```

Comment:
Youtube attribution is "HARSHVARDHAN RANA"

Reply:
Hello @eshellman, I wanted to let you know that the course consists of a single 7-hour long video without a playlist. So, I took the initiative to create one myself. However, I came across an even better course that already has a playlist curated by the creator. I've integrated it into the code.
Regards

Reply:
Aha. A 7-hour single video is fine, as long as you tell us that in the description of the PR (guideline is >1 hr.)

Reply:
you should switch it back to what you had before. sorry for the confusion.

Reply:
Hello @eshellman, I've re-written that course and also included the one by the same creator that I found to be more beneficial in the code.